docs/howto/bootstrap-taskgraph.rst

```diff
@@ -4,13 +4,74 @@ Bootstrap Taskgraph in Decision Task
 This how-to guide outlines the recommended way to bootstrap Taskgraph (and
 other task generation dependencies) in a :term:`Decision Task`.
 
-If you'd like to follow along, you can download the example
-:download:`.taskcluster.yml </tutorials/example-taskcluster.yml>` file.
+Using the Pre-Built Docker Images
+---------------------------------
+
+The simplest way to make Taskgraph available to your Decision task is to use
+Taskgraph's pre-built Docker images under the `mozillareleases/taskgraph`_ Docker Hub
+repository.
+
+In your Decision task definition in ``.taskcluster.yml``, add the following to the task's
+``payload`` section:
+
+.. code-block:: yaml
+
+   payload:
+     image: mozillareleases/taskgraph:decision-latest
+
+This will ensure your Decision task always run with the latest version of Taskgraph.
+Of course, it's best practice to pin your CI dependencies, so it's recommended to pin
+to a specific version:
+
+.. code-block:: yaml
+
+   payload:
+     image: mozillareleases/taskgraph:decision-v7.3.0
+
+While not required, it's also best practice to specify the image hash:
+
+.. code-block:: yaml
+
+   payload:
+     image: mozillareleases/taskgraph:decision-v7.3.0@sha256:f954e54d427d08b18954a4ff4541d43c76c476d0738f341e4fa82a959939d3fa
+
+The hash can be found on the Docker Hub page for the specific tag you are
+pinning to.
+
+.. _mozillareleases/taskgraph: https://hub.docker.com/r/mozillareleases/taskgraph/tags
+
+
+Using a Python Requirements File
+--------------------------------
```

Comment:
Nit: Looks like you're using ~~~~ down below, but ----- here, is that intentional?

Reply:
Yeah, that makes it a smaller heading in rST